ledger/participant-state/kvutils/tools/src/main/scala/ledger/participant/state/kvutils/tools/integritycheck/Config.scala

```diff
@@ -12,6 +12,9 @@ case class Config(
     exportFilePath: Path,
     performByteComparison: Boolean,
     sortWriteSet: Boolean,
+    indexOnly: Boolean = false,
+    fullMetrics: Boolean = false,
```

Comment:
Please make this consistent with the command-line option itself.

Reply:
Done.